packages/ui-app/src/util/isValidBalance.spec.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { BIT_LENGTH_128 } from '../constants';
+import isValidBalance from './isValidBalance';
+import translate from '../translate';
```

Comment:
(nitpick) move translate one up, so external to folder first, then internal to folder.